src/liblibc/lib.rs

```diff
@@ -3912,6 +3912,9 @@ pub mod consts {
             pub const _SC_XBS5_ILP32_OFFBIG : c_int = 126;
             pub const _SC_XBS5_LPBIG_OFFBIG : c_int = 128;
 
+            // Linux
+            pub const _PC_NAME_MAX: c_int = 3;
+            pub const _PC_PATH_MAX: c_int = 4;
```

Comment:
Currently I placed the constants in the `sysconf` module, as `pathconf` seemed to be categorized as the same as `sysconf. Do you believe this is the right place?

After all, the entire structure of `libc` seems to need some refactoring, I guess...